src/main/java/module8/Task3.java

```diff
@@ -0,0 +1,35 @@
+package module8;
+/*Задание 3
+Используйте классы задания 2 модуля 3 для реализации задачи поиска человека по паспортным данным.
+Предусмотрите новый класс Passport в котором будут серия и номер паспорта (серия типа String, номер типа Integer).
+Создайте несколько экземпляров Passport и Person для установления связи между ними используйте HashMap<Passport, Person>
+Для поиска создайте новый экземпляр Passport в который внесите заведомо существующие данные (совпадающие с данными
+Passport на предыдущем шаге), проверьте поведение приложения если паспортные данные внесены с ошибкой.
+Обратите внимание на поведение HashMap в случае если Passport не имеет методов hashCode и equals
+*/
+
+import java.util.HashMap;
+
+public class Task3 {
+    public static void main(String[] args) {
+        HashMap<Passport, Person> map = new HashMap<>();
```

Comment:
`Map<Passport, Person> map = new HashMap<>();`

Reply:
Done. HashMap was changed to Map